.pre-commit-config.yaml

```diff
@@ -45,3 +45,15 @@ repos:
     rev: v0.0.9  # Use the ref you want to point at
     hooks:
     -   id: add-msg-issue-prefix
+-  repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v4.0.0-alpha.8
+    hooks:
+    -  id: prettier
+       files: \.(js|css|html|vue)$
```

Comment:
let's add typescript (.ts) and react (.jsx) files as well in case we use them in the future